alws/models.py

```diff
@@ -101,12 +101,31 @@ class Platform(Base):
         unique=True,
         index=True
     )
+    reference_platform = sqlalchemy.Column(
+        sqlalchemy.VARCHAR(length=20), nullable=True)
```

Comment:
It seems to me, what it can be more than one reference_platform, so it's better to be many-to-many relationship

Reply:
Thanks, fixed